gdal/doc/source/tutorials/geotransforms_tut.rst

```diff
@@ -0,0 +1,28 @@
+.. _geotransforms_tut:
+
+================================================================================
+Geotransform Tutorial
+================================================================================
+
+Introduction to Geotransforms:
+-------------------------------
+
```

Comment:
A geotransform is an affine transformation from the image coordinate space (row, column) to the georeferenced coordinate space (projected or geographic coordinates). It consists in a set of 6 coefficients:

Reply:
added